src/main/java/org/jclouds/vcloud/director/v1_5/domain/dmtf/Envelope.java

```diff
@@ -81,6 +85,6 @@ public ReferencesType getReferences() {
    }
 
    public VirtualSystem getVirtualSystem() {
-      return virtualSystem;
+      return virtualSystem != null ? virtualSystem : virtualSystemCollection.getVirtualSystems().iterator().next();
```

Comment:
@aledsage +1
also I'm not totally sure we should get the first VirtualSystem. Do they all contain the same types? or we can filter more accurately?